src/modules/task-collection.js

```diff
@@ -46,18 +62,35 @@ export default class TaskCollection {
       return task;
     });
     localStorage.setItem('tasks', JSON.stringify(this.tasks)); // save updated collection to localStorage
-  };
+  }
+
+  clearCompletedTask() {
+    const number = this.tasks.filter((task) => task.completed === true).length;
+    this.count -= number;
 
-  displayTasks = () => {
+    this.tasks = this.tasks.filter((task) => task.completed !== true);
+    this.tasks.forEach((task, i) => {
+      task.index = i + 1;
+    });
+
+    localStorage.setItem('tasks', JSON.stringify(this.tasks));
+    this.displayTasks();
+  }
```

Comment:
- [ ] Please according to the project requirement, kindly create a separate file and import it as a module. It should contain methods related to the **status updates** and **clear all completed**. ✔️